metron-platform/metron-enrichment/README.md

```diff
@@ -76,6 +62,19 @@ intel bolt, the configurations will be taken from the respective join bolt
 parallelism.  When proper ambari support for this is added, we will add
 its own property.
 
+### Split-Join Enrichment Topology
+
+The now-deprecated split/join topology is also available and performs enrichments in parallel.
+This poses some issues in terms of ease of tuning and reasoning about performance.
+
+![Architecture](enrichment_arch.png)
+
+#### Using It
+
+In order to use the older, deprecated topology, you will need to
+* Edit `$METRON_HOME/bin/start_enrichment_topology.sh` and adjust it to use `remote-splitjoin.yaml` instead of `remote-unified.yaml`
```

Comment:
Instead of saying you need to edit the script, we could just give them the following command.  You just need to pass in the right command line args to use Split/Join.
```
$METRON_HOME/bin/start_enrichment_topology.sh --remote $METRON_HOME/flux/enrichment/remote-splitjoin.yaml --filter $METRON_HOME/config/enrichment-splitjoin.properties
```

Reply:
This actually wasn't net-new, it's the way we currently expose the choice. I would prefer to keep it as-is and manual because the desire is NOT to have customers use the deprecated topology.

Reply:
Nevermind, just re-read that suggestion. Sounds good to me @nickwallen.